examples/floor-plan/index.ts

```diff
@@ -0,0 +1,79 @@
+// Shims
+import 'reflect-metadata';
+import 'zone.js';
+
+import { YagaModule } from '../../lib/index'; // @yaga/leflet-ng2
+
+import { Component, PlatformRef } from '@angular/core';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { CRS, latLng, LatLngBounds } from 'leaflet';
+
+import { ExampleAppComponentBlueprint, IExampleProperties } from '../app-component-blueprint';
+import { ExamplePropertiesModule, PROPERTIES_WRAPPER } from '../property.component';
+
+const platform: PlatformRef = platformBrowserDynamic();
+
+/* tslint:disable:max-line-length */
+const template: string = `
+<example-header [title]="'Floor Plan'"></example-header>
+<div class="container">
+  <div class="map">
+    <yaga-map
+          [(zoom)]="getDuplexPropertyByName('zoom').value"
+          [(lat)]="getDuplexPropertyByName('lat').value"
+          [(lng)]="getDuplexPropertyByName('lng').value"
+          [(minZoom)]="getDuplexPropertyByName('minZoom').value"
+          [(maxZoom)]="getDuplexPropertyByName('maxZoom').value"
+          [(maxBounds)]="getDuplexPropertyByName('maxBounds').value"
+          [crs]="crs">
+      <yaga-image-overlay
+        [url]="'https://dl.dropbox.com/s/yhrpnftsuis15z6/Topkapi_Palace_plan.svg'"
+        [bounds]="getDuplexPropertyByName('maxBounds').value">
+      </yaga-image-overlay>
+    </yaga-map>
+  </div>
+  <p>Inspiration and the floor map taken from here: <a href="https://codepen.io/zachdunn/pen/VvRXdP">https://codepen.io/zachdunn/pen/VvRXdP</a></p>
```

Comment:
I would prefer `<a href="..." target="_blank">...</a>`... But I will add an issue for that...